src/js/tabs/pid_tuning.js

```diff
@@ -1842,7 +1860,7 @@ TABS.pid_tuning.initialize = function (callback) {
                 } else {
                     slider.attr('step', SLIDER_STEP_UPPER);
                 }
-                if (!TuningSliders.expertMode) {
+                if (!TuningSliders.expertMode && !semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
```

Comment:
```suggestion
                if (!TuningSliders.expertMode && semver.lt(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
```
Not sure what's preferred.

Reply:
Agreed.  I just learned about gte vs lte vs gt vs lt.  :-)